tardis/model/base.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+from astropy import constants, units as u
+
+from tardis.model import density
+from tardis.util import quantity_linspace, InstanceDescriptorMixin
+
+
+class Radial1DModel(InstanceDescriptorMixin):
+    def __init__(self, velocity, homologous_density, abundance, t_inner,
+                 time_explosion, t_radiative=None, dilution_factor=None,
+                 v_boundary_inner=None, v_boundary_outer=None):
+        self.velocity = velocity
+        self.homologous_density = homologous_density
+        self.abundance = abundance
+        self.t_inner = t_inner
+        self.time_explosion = time_explosion
+
+        if t_radiative is None:
+            lambda_wien_inner = constants.b_wien / self.t_inner
+            self.t_radiative = constants.b_wien / (lambda_wien_inner * (
+                1 + (self.v_middle - self.v_inner[0]) / constants.c))
+        else:
+            self.t_radiative = t_radiative
+
+        if dilution_factor is None:
+            self.dilution_factor = 0.5 * (1 - np.sqrt(
+                1 - (self.r_inner[0] ** 2 / self.r_middle ** 2).to(1).value))
```

Comment:
This is the geometric_w. @wkerzendorf  Do you think we should have it as a property? Would it ever be accessed again except initializing the w?